monitoring/grafana.service

```diff
@@ -4,8 +4,8 @@ After=network.target
 
 [Service]
 Type=simple
-WorkingDirectory=/usr/bin/piman/monitoring
-ExecStart=/usr/bin/piman/monitoring/grafana.sh
+WorkingDirectory=/home/usr/piman/monitoring
```

Comment:
we need to agree on a location for this code! we have conflicting ideas. can you start a discussion?

Reply:
A Canvas Discussion, right?